### ISSUE TITLE
Cleanup Take-home

### ISSUE THREAD

**Issue by leomaxi** [open-learning-exchange/take-home#45]
Delete and remove all old/unused classes,images and XML files

**Comment by timecomplexity** [comment]
I am currently working on this issue and I wanted to make sure of a few things. Thus far, I have deleted the unused image files (of which there were many) and I have run Lint to help check for unused files and classes.

I have discovered these different types of classes listed below. I am not sure whether or not to delete some of them, as some of them appear to be works in progress or placeholder, empty classes. Lint gave me messages saying that all of these classes are not instantiated

(I have placed a comment and the file name it is located in in _italics_):

**Two classes with the same name in different java files:**

- TestAsyncPush[_SyncDevice.java_ - Appears to be a test class]
- TestAsyncPush [_NewSync.java_]

**Works in progress?**

- SyncAllResource [_SyncAllResource.java_ - Completely commented out. WIP?]
- ProcessSync [_ProcessSync.java_ - Appears to be a WIP]
- EditMovieTask [_EditMovieTask.java_ - WIP?]

**Empty Classes:**

- AsyncCheckConnection [_AsyncCheckConnection.java_ - Empty Class]
- SyncResource [_SyncResource.java_ - Empty class]

And in regards to the XML files, I don't seem to be coming up with any that are "unused", but I am finding XML tags with empty bodies and XML schemas that are declared but are not used. Would it be safe to assume that these schemas and empty bodies are no longer required and go ahead and delete them?
